### ISSUE TITLE
Choose context menu to show

### ISSUE THREAD

**Issue by khabib227** [YoshifumiFuyuno/Search-from-Popup-or-ContextMenu#71]
### Is your feature request related to a problem? Please describe.

For instance just show tab context menu, or just for images, currently we have it per engine, it is excellent , but not has a way to disable each context menu for instance:

Show context menu for images (x)
Show context menu for tab ( )

etc....

### Describe the solution you'd like

.

### Describe alternatives you've considered

.

**Comment by YoshifumiFuyuno** [comment]
Hi.
Thank you for your suggestion.

Do you mean that you want to switch enable/disable all at once on the option page?
Sorry, I'm very busy this month, so I won't be able to add any features until next month.

Best regards

**Comment by khabib227** [comment]
Not exactly. Currently we have checkboxes to choose where we want to show the engines, so I can show Bing on Tab Context Menu but not show in Image Context Menu, but if I want disable completely the Tab context menu, if I want hide it, keeping the others context menus? We haven't a option to it currently, we just can disable all context menus together.

**Comment by YoshifumiFuyuno** [comment]
Do you mean you want to change all the check marks for "images", "tabs", etc. at once?

**Comment by khabib227** [comment]
Yes or a checkbox to switch each context menu definitively, to set the default state, so if I disable for instance the selection context menu, if I add a new engine this context menu would be disabled by default. Thanks the attention!

**Comment by YoshifumiFuyuno** [comment]
I see, I understand. Thanks for explaining.
I think it's a good idea!
